charter.md

```diff
@@ -0,0 +1,104 @@
+# SIG Release Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+- Production of high quality Kubernetes releases on a reliable schedule
+- Ensure there is a consistent group of community members in place to support the release process
+- Provide guidance and tooling to facilitate the production of automated releases
+- Serve as a tightly integrated partner with other SIGs to empower SIGs to integrate their repositories into the release process
+
+### In scope
+
+- Ensuring high quality Kubernetes releases
+  - Define and staff release roles to manage the resolution of release blocking criteria
+  - Define and drive development processes (e.g. merge queues, cherrypicks) and release processes 
+    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
+  - Manage the creation of release specific artifacts including
+    - Code branches
+    - Binary distributions
+    - Release notes
+- Continually improving release and development processes
+  - Work closely with SIG Contributor Experience to define and build tools to facilitate release process (e.g. dashboards)
+  - Work with downstream communities responsible for packaging Kubernetes releases
+  - Work with other SIGs to agree upon the responsibilities of their SIG with respect to the release
+  - Define and collect metrics related to the release in order to measure progress over each release
+- Collaboration with downstream communities which build artifacts from Kubernetes releases
+  - Ensure the appropriate level of integration with publishing the OSS build artifacts
+
```

Comment:
Conduct release retrospectives

Reply:
Hmmm, I'm thinking this [sh|c]ould be a function of SIG PM. I'll add language that it should be facilitated by both.

Reply:
Added.